views/dashboard.handlebars

```diff
@@ -0,0 +1,18 @@
+{{!-- EXAMPLE of linking to tailwind.css styles
+
+<!doctype html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
+  ## LINK to the output.css file to get access to styles
+  <link href="../dist/output.css" rel="stylesheet">
```

Comment:
This is just an example of how to link the tailwind styles to our .handlebars files